dirac.cfg

```diff
@@ -419,6 +419,94 @@ Systems
 Resources
 {
 
+  # Section for setting options for ComputingElements
+  Computing
+  {
+
+    # ComputingElement options can be set with different degrees of specialization:
+    #
+    # /Resources/Computing/CEDefaults* for all computing elements
+    # /Resources/Computing/<CEType>* for CEs of a given type, e.g., HTCondorCE or ARC
+    # /Resources/Sites/<grid>/<site>/CEs* for all CEs at a given site
+    # /Resources/Sites/<grid>/<site>/CEs/<CEName>* for a specific CE
+    #
+    # Values are overwritten by the most specialized option.
+
+    # The options below can be valid for all computing element types
+    CEDefaults
+    {
+
+      # Default environment file sourced before calling rid commands, without extension '.sh'
+      GridEnv = /opt/dirac/gridenv
+
+      # Will be added to the pilot configuration as /LocalSite/SharedArea
+      SharedArea = /cvmfs/lhcb.cern.ch/lib
+
+      # or adding some generic pilot options (only for pilots submitted by SiteDirectors)
+      # the example below will add the environment variable DIRACSYSCONFIG (see :ref:`bashrc_variables`)
+      ExtraPilotOptions = --userEnvVariables DIRACSYSCONFIG:::pilot.cfg
+
+      # for adding the --modules=value option to dirac-pilot
+      Modules =
+
+      # for adding the --pipInstallOptions=value to dirac-pilot
+      PipInstallOptions = --index-url https://lhcb-repository.web.cern.ch/repository/pypi/simple
```

Comment:
@atsareg I'm pretty sure you can use this option to add COMDIRAC to Python 3 pilots